### ISSUE TITLE
read_chr_bed error

### ISSUE THREAD

**Issue by rsalz** [ekawaler/pyQUILTS#21]
I already prepared the proteome and genome as instructed and tried to run quilts.py with the following command:

python2 quilts.py --output_dir ../databases/ --junction ../junctions/two_pass/test/ --junction_file_type star --germline ../vcf/test/ --genome genome/hg38/ --proteome proteome/

and then i get this error immediately, with no further explanation:

/bin/sh: 1: /mnt/xomics/renees/data/recognition/pyQUILTS/read_chr_bed: Exec format error
ERROR: read_chr_bed didn't work - now we don't have a proteome.bed.dna file.
Aborting program.

What's going on? why is it stopping

**Comment by ekawaler** [comment]
It seems your computer is having trouble running the C file. Have you tried recompiling read_chr_bed?

**Comment by rsalz** [comment]
right, that error is fixed now. but now when I run it i get "Error opening: genome/hg38/chr6.fa.cmp1" for all .cmp1 files in the genome

**Comment by ekawaler** [comment]
Did you check to see whether those files exist at that location?

**Comment by rsalz** [comment]
ah yes. they are named something else in the folder. the naming convention in my genome folder is 'Homo_sapiens.GRCh38.dna_rm.chromosome.21.fa.cmp1' while the quilts script looks for chr21.fa.cmp1. strange that this wasn't correct despite me following the steps in 'prepare_genome'. i'll rename the files and try again...

**Comment by rsalz** [comment]
now it found other files it can't open:
'Error opening: genome/hg38/chrCHR_HG1342_HG2282_PATCH.fa.cmp1'
Is it possible the 'prepare_genome' portion isn't really preparing my genome for running quilts.py?

**Comment by ekawaler** [comment]
There are often some non-canonical chromosomes in the variant sets - you can usually safely ignore those errors. The vast majority of your variants should still come out fine.
